storage.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/net/context"
+)
+
+var (
+	client     *storage.Client
+	bucket     *storage.BucketHandle
+	bucketName string
+)
+
+func init() {
+	var err error
+	client, err = storage.NewClient(context.Background())
+	if err != nil {
+		panic(err)
+	}
+
+	bucketName = os.Getenv("SITE_BUCKET")
+	if bucketName == "" {
+		panic("No SITE_BUCKET given")
+	}
+}
+
+func handleStaticRequest(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Served-With-Love-By", "Gophers")
+
+	bucket := client.Bucket(bucketName)
+
+	path := r.URL.Path[1:] // removing the leading / here
+
+	if path == "" {
+		path = "index.html"
+	}
+
+	if path[:7] == "assets/" {
+		w.Header().Set("Cache-Control", "max-age=86400")
+	}
+
+	object := bucket.Object(path)
```

Comment:
oo i see what you are doing here..

so notice that each time someone hits womenwhogo.org (the way this is written now), there are 2 hops: 1 to get the html or whatever from GCS, and then one to get it back to the end user.

it would be more efficient if we just stored the static stuff on the box, rite? then it's just one hop each time!

I recommend instead using a `http.FileServer`. Take a look at this tutorial for doing it on appengine: https://cloud.google.com/appengine/docs/flexible/go/serving-static-files

but you'd need to wrap it, b/c we want that awesome cache header you set.
so it would be like:

```
func assetHandler(w http.ResponseWriter, r *http.Request) {
    w.Header().Set("Cache-Control", "max-age=86400")
    http.FileServer(http.Dir("/PATH/TO/FILES"))(w, r)
}
```